rebuild.sh

```diff
@@ -13,6 +13,20 @@ else
     echo "Shortcut change already confirmed"
 fi
 
+# Ensure TypeScript is new enough, offer to install with npm if not
+export PATH="./node_modules/.bin:$PATH"
```

Comment:
Do we want to install this in the project folder? Shouldn't it be installed user-wide?

Reply:
I think it depends on the `prefix` (.e.g /usr/bin/node) for `npm install -g`. `npm install` will guarantee that the user will not need sudo.

Reply:
@mmstick I personally think that it is the cleanest to install it in the project where it can be trivially cleaned up with `git clean` once Pop Shell has been successfully installed. This is also the default behaviour of `npm install`, so it is the expected behaviour.

If we instead install it somewhere else within the user's `$HOME` means we're polluting it in perpetuity. A global `npm -g`install would, as @jmmaranan points out, require superuser privileges so I consider that approach out of the question.